src/functionalTest/resources/features/common/mca/Check_NoC_Approval__Base.td.json

```diff
@@ -0,0 +1,16 @@
+{
+  "_guid_": "Check_NoC_Approval__Base",
+
+  "productName": "Manage Case Assignment Microservice",
+  "operationName": "Check NoC Approval",
+
+  "method": "POST",
+  "uri": "/noc/check-noc-approval",
+
+  "s2sClientId": "xui_webapp",
```

Comment:
If mimicking data-store calling your callback: should `s2sClientId` not be `"ccd_data"`?

NB: you may need to add `ccd_data` equivalents to the following to make this work:
* befta env vars: https://github.com/hmcts/aac-manage-case-assignment/blob/ACA-85/aca-docker/bin/env_variables_all.txt#L6-L7
* jenkins updates:
   * S2S Key: https://github.com/hmcts/aac-manage-case-assignment/blob/ACA-85/Jenkinsfile_CNP#L39
   * S2S ID: https://github.com/hmcts/aac-manage-case-assignment/blob/ACA-85/Jenkinsfile_CNP#L66

NB: Although updates to S2S key in docker config is not required: https://github.com/hmcts/ccd-docker/blob/master/compose/backend.yml#L122

Reply:
Ideally yes but MCA endpoint code doesn't care whether it's ccd_data or xui_webapp.

Reply:
Agreed but using the correct code will allow us to test that your `MANAGE_CASE_S2S_AUTHORISED_SERVICES` change has been preserved and applied to the MCA instance.